src/arithmetic/time_funcs/time_funcs.h

```diff
@@ -0,0 +1,11 @@
+/*
+* Copyright 2018-2019 Redis Labs Ltd. and Contributors
+*
+* This file is available under the Redis Labs Source Available License Agreement
+*/
+
+#pragma once
+
+#include "../../value.h"
+
+void Register_TimeFuncs();
```

Comment:
Missing new line at EOF

Reply:
My autoformatting on save removes newlines at EOF (though that must be specific to the plugin I'm using, since `make format` does not). I'll try to tweak that!